docs/Getting_started.md

```diff
@@ -57,6 +57,38 @@ npx mrm license --config:name "Gandalf the Grey" --config:email "gandalf@middlee
 
 ## Configuration
 
+There are three ways for setting tasks configurations: **interactive mode**; **command line parameters**; **config files**.
+
+> See [tasks docs](../Readme.md#tasks) for available config options on core tasks.
+
+### Interactive mode
+
+Use the `-i` argument to enable interactive (prompt) mode for configs:
```

Comment:
We use `config` to refer to a config file, not to particular options. It looks confusing here.